st2common/st2common/router.py

```diff
@@ -513,6 +514,10 @@ def __init__(self, **entries):
 
         try:
             resp = func(**kw)
+        except DataStoreKeyNotFoundError as e:
+            LOG.warning('Failed to call controller function "%s" for operation "%s": %s, %s' %
```

Comment:
Can you please also show an example how the new Warning looks like in the logs?

Reply:
Ok. Will do.